package.nls.json

```
@@ -6,14 +6,16 @@
     "containerApps.enableOutputTimestamps": "Prepends each line displayed in the output channel with a timestamp.",
     "containerApps.browse": "Browse",
     "containerApps.createContainerApp": "Create Container App...",
-    "containerApps.deployImage": "Update Container App Image......",
+    "containerApps.editContainerApp": "Edit Container App (Advanced)",
```

```suggestion
    "containerApps.editContainerApp": "Edit Container App (Advanced)...",
```

As there are more steps for editing after clicking.

---

Nope, not unless it's command palette initiated and you need to pick the container app

---

Doesn't it open the json file and then you have to make edits for it to actually do anything though?

---

Oh does that count as two? In that case yes.  There's no wizard steps or anything though

---

Yeah, I would say that because opening the file isn't actually editing it, and in order to edit it, you need to do more steps, that this is a `...` command.

Convoluted? Maybe. But so is this command 🤣

---

Well, it is `advanced`™️